x-pack/docs/en/ilm/apis/set-policy.asciidoc

```diff
@@ -0,0 +1,96 @@
+[role="xpack"]
+[testenv="basic"]
+[[ilm-put-policy]]
+=== Set Policy On Index API
+++++
+<titleabbrev>Set Policy On Index</titleabbrev>
+++++
+
+Assigns a policy to an index for management.
+
+==== Request
+
+`POST <index>/_ilm/<policy>`
+
+==== Description
+
+This action assigns a policy to an index. It is effectively the same as setting an index's
+`index.lifecycle.name` setting.
+
+==== Path Parameters
+
+`index` (required)::
+  (string) Identifier for the index.
+
+`policy` (required)::
+  (string) Identifier for the policy.
+
+==== Request Parameters
```

Comment:
This API also supports the indices options like `allow_no_indices` and `wildcard_expand_open` etc, that should be mentioned here

Reply:
For other index APIs we don't seem to have these options repeated in each API but instead have an API conventions page to describe the multiple indices and index options (https://www.elastic.co/guide/en/elasticsearch/reference/6.4/multi-index.html). I wonder if we should just add a link on the `index` parameter explaining that multiple index options apply as described on that page of the docs rather than repeating the actual options everywhere?